### ISSUE TITLE
Implement latest notification specification and webhooks

### ISSUE THREAD

**Issue by bourgeoa** [nodeSolidServer/node-solid-server#1722]
Help needed

**Comment by csarven** [comment]
If this issue is to show intent to implement some of notifications specifications, can you please update the table in https://github.com/solid/notifications/issues/141#issue-1526240333 or let me know so I can.

re the issue title, want to doublecheck whether you intended to write Webhook (and/)or Websocket notification channel type? (Vaguely recall a past discussion on it being about Websocket).

Solid Notifications Protocol Editor's draft: https://solid.github.io/notifications/protocol
Solid Notifications Protocol Latest published version: https://solidproject.org/TR/notifications-protocol
Notification Channel Type registry: https://solidproject.org/TR/#notification-channel-type-registry

**Comment by TallTed** [comment]
Better not to reference "latest" anything, significantly but not only because that may change from opening to closing of issue and/or PR.

Better to instead refer to specific version and/or date of external specs, docs, etc. I strongly advise updating the title of this issue as described.

**Comment by csarven** [comment]
I agree. I shared latest for quick navigation (general purpose communication). Not intended to suggest that latest URL should be implemented.

What's recommended elsewhere (in discussions, QA document) is to refer to specific versions when need to be clear about conformance.

Implementers are also encouraged to use specific URIs in code comments or in other communication. For testing and reporting purposes, generally versioned is preferred, but it could be anything - I'd suggest either a published versioned URI or an editor's draft URI. It depends on how the reports are purposed. Versioned would be for wide communication whereas the ED would be more local/internal communication or experimental/WIP.

I've updated my earlier comment to include the versioned URI.

**Comment by bourgeoa** [comment]
A few links to help anyone interested in this issue :

- Actual insecure implementation in NSS :
  - https://github.com/nodeSolidServer/node-solid-server/blob/958a6ef82e67c5bfc000488aca4c8bfdd6a9bfa0/lib/create-server.js#L7
  - https://github.com/nodeSolidServer/node-solid-server/blob/958a6ef82e67c5bfc000488aca4c8bfdd6a9bfa0/lib/handlers/get.js#L32
  - https://github.com/nodeSolidServer/node-solid-server/blob/958a6ef82e67c5bfc000488aca4c8bfdd6a9bfa0/lib/ldp.js#L336
  
- In CSS https://github.com/CommunitySolidServer/CommunitySolidServer/tree/main/src/server/notifications and in CSS documentation
